From: Michael Catanzaro <mcatanzaro@gnome.org>
Date: Mon, 21 Nov 2022 17:49:29 +0000
Subject: M!108 - Try harder to support webkitgtk-6.0

The cdb16f26f63f5093479a43cab32012845bcf33ed attempted to handle WebKitGTK
API changes, but I forgot the header file name changed as well.

Closes https://gitlab.gnome.org/GNOME/evolution-data-server/-/merge_requests/108

(cherry picked from commit 8d29b0bded2e4e3700ce08e9db2eaeac4400528a)
---
 src/libedataserverui/e-credentials-prompter-impl-oauth2.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/libedataserverui/e-credentials-prompter-impl-oauth2.c b/src/libedataserverui/e-credentials-prompter-impl-oauth2.c
index 289d0c6..56483ac 100644
--- a/src/libedataserverui/e-credentials-prompter-impl-oauth2.c
+++ b/src/libedataserverui/e-credentials-prompter-impl-oauth2.c
@@ -32,21 +32,19 @@
 #if GTK_CHECK_VERSION(4, 0, 0)
 #ifdef ENABLE_OAUTH2_WEBKITGTK4
 #define WITH_WEBKITGTK 1
+#include <webkit/webkit.h>
 #else
 #undef WITH_WEBKITGTK
 #endif
 #else
 #ifdef ENABLE_OAUTH2_WEBKITGTK
 #define WITH_WEBKITGTK 1
+#include <webkit2/webkit2.h>
 #else
 #undef WITH_WEBKITGTK
 #endif
 #endif
 
-#ifdef WITH_WEBKITGTK
-#include <webkit2/webkit2.h>
-#endif /* WITH_WEBKITGTK */
-
 struct _ECredentialsPrompterImplOAuth2Private {
 	GMutex property_lock;
 
